packages/node/src/integrations/http/SentryHttpInstrumentation.ts

```diff
@@ -69,6 +69,22 @@ export type SentryHttpInstrumentationOptions = InstrumentationConfig & {
    */
   ignoreIncomingRequestBody?: (url: string, request: http.RequestOptions) => boolean;
 
+  /**
+   * Controls the maximum size of HTTP request bodies attached to events.
+   *
+   * Available options:
+   * - 'none': No request bodies will be attached
+   * - 'small': Request bodies up to 1,000 bytes will be attached
+   * - 'medium': Request bodies up to 10,000 bytes will be attached (default)
+   * - 'always': Request bodies will always be attached
+   *
+   * Note that even with 'always' setting, bodies exceeding 1MB will never be attached
+   * for performance and security reasons.
+   *
+   * @default 'medium'
+   */
+  maxRequestBodySize?: 'none' | 'small' | 'medium' | 'always';
```

Comment:
should we rename this to `maxIncomingRequestBodySize`? It is not clear right now that this only applies to incoming requests, not outgoing ones 😅

Reply:
I got the name from the general Sentry docs: https://develop.sentry.dev/sdk/expected-features/#attaching-request-body-in-server-sdks

To be uniform with other SDKs, it would be beneficial to keep the name, but I agree that it would be clearer with `incoming`. At least, the information is available in the JSDoc 😅

Reply:
we don't _need_ to have the same name, if we think something else is better :) maybe you could also check with other sdk folks, if this also applies to outgoing requests in other implementations or just to incoming?